.nvmrc

```diff
@@ -0,0 +1 @@
+v15.5.1
```

Comment:
hum :D

Reply:
would this be a problem in the future?

i'll test it with 12 (this seems to be the team's node version) if its needed

Reply:
14 is better, lts

Reply:
done